### PR TITLE
Updated to_hit

### PR DESCRIPTION
**Pull request by TealcOneill**
#### Summary
Infrastructure "All the melee weapons now use the new to_hit format."

#### Purpose of change
Makes it so future contributors don't mistakenly copy the old to_hit format. 

#### Describe the solution
Replaces the to_hit int in all items under the melee items folder with the new string value thing.

#### Testing
Loaded up to make sure I didn't accidentally change any values.

#### Additional context
I've found quite a few items that had weird to_hit values but I didn't want to start changing them randomly in this pr so if and when this one gets merged I'll go back in with my notes and make a new pr with all those changes.

### CODE REVIEW

**Comment by TealcOneill** [comment]
Damn, I got off on a few. I'll re go through and fix these when I'm not about to pass out.

File: data/json/items/melee/misc.json [hunk 1]
```diff
@@ -77,4 +78,4 @@
       { "pocket_type": "MAGAZINE_WELL", "flag_restriction": [ "BATTERY_MEDIUM" ], "default_magazine": "medium_battery_cell" }
     ]
   }
-]
+]
```
Comment (Maleclypse) [inline data/json/items/melee/misc.json:81]:
```suggestion
]

```

**Comment by Maleclypse** [comment]
It showed there was an end of file error and so I've tried to fix that but I'm not sure why linting didn't fix it.  We'll see if this passes the tests.

**Comment by TealcOneill** [comment]
Thank you!

**Pull request merged**